### ISSUE TITLE
fetch(request) fails

### ISSUE THREAD

**Issue by jakearchibald** [github/fetch#122]
```js
var request = new Request('/hello');
fetch(request);
```

…results in fetching `/[object%20Object]`

**Comment by mislav** [comment]
This is not yet supported. See #116.

In the meantime, this could work:

```js
new Request('/hello').fetch()
```

**Comment by dgraham** [comment]
We need to make `Request#fetch` private. It is not public API.

**Comment by IPRIT** [comment]
![art- - -fantasy- -1308038](https://cloud.githubusercontent.com/assets/1553519/8550032/bec07538-24d6-11e5-9337-b06358b7b28a.jpeg)

**Comment by dgraham** [comment]
Fixed in ab1ac93.
